src/data/ContributionResources/Git.jsx

```diff
@@ -1,22 +1,19 @@
-import ContributionResources from ".";
-
 export default {
-    'user-responses': [
-      {
-        name: 'How to work with Git?',
-        link: 'http://learngitbranching.js.org/',
-      },
-      {
-        name: 'How to work with forks?',
-        link: 'https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/working-with-forks'
-      },
-      {
-        name: 'Done. What else should I know about?',
-        child: ContributionResources,
-      },
-    ],
-    'gooey-response': {
-      gooey: 'New to Git / GitHub? Check this out.',
+  'user-responses': [
+    {
+      name: 'How to work with Git?',
+      link: 'http://learngitbranching.js.org/',
     },
-  };
-  
\ No newline at end of file
+    {
+      name: 'How to work with forks?',
+      link: 'https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/working-with-forks',
+    },
+    {
+      name: 'Done. What else should I know about?',
+      jump: 'u0u2u0u0',
```

Comment:
This particular component was causing a circular dependency problem that crashed the app on startup as it referred to the `ContributionResources` which in turn referred back to this `Git` component. I've fixed this by replacing the 'child' param with the equivalent 'jump' param 'pathId' value that points to the same expected page